app/jobs/reindex_by_druid_job.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Reindexes an object given a druid
+class ReindexByDruidJob
+  include Sneakers::Worker
+  # This worker will connect to "dor.indexing-with-model" queue
+  # env is set to nil since by default the actual queue name would be
+  # "dor.indexing-with-model_development"
+  from_queue 'dor.indexing-by-druid', env: nil
```

Comment:
Should this comment block have `s/with-model/by-druid/g`?

Reply:
Yep, I've made this fix.